P/Prjtrellis/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Prjtrellis"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/prjtrellis.git", "dff1cbcb1bd30de7e96f8a059f2e19be1bb2e44d")
+]
+
+dependencies = [
+    Dependency("Python_jll"),
+    Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd prjtrellis/libtrellis
+cmake -DCMAKE_INSTALL_PREFIX=${prefix}
```

Comment:
```suggestion
mkdir build && cd build
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    ..
```

Reply:
Why add a `build` dir?

Reply:
ah I see the `..` at the end.

Reply:
When something fails and you want to restart from scratch, there is no way to do that with CMake unless you do a build in a subdirectory can you completely and safely delete